packages/iTwinUI-react/src/core/Table/cells/EditableCell.tsx

```diff
@@ -41,7 +40,15 @@ export type EditableCellProps<
 export const EditableCell = <T extends Record<string, unknown>>(
   props: EditableCellProps<T>,
 ) => {
-  const { cellElementProps, cellProps, onCellEdit, children, ...rest } = props;
+  const {
+    cellElementProps,
+    cellProps,
+    onCellEdit,
+    children,
+    isDisabled,
+    ...rest
+  } = props;
+  isDisabled; // To omit and prevent eslint error.
```

Comment:
Cant we omit this prop before it even reaches editable cell? Looks weird now..

Reply:
You want to ask our users not to pass `{...props}`? 😀
![image](https://user-images.githubusercontent.com/36186912/166440868-9a93842d-195c-47fb-86d4-e700bdabdcc1.png)

Reply:
ahh... thats weird..
Cant editable cell be disabled, afterall? :D

Reply:
Then it would be just a default cell, doesn't make sense.

Reply:
Thats fine. Users would not need to switch between cells in their custom implementations.
They just pass all the props and be calm about it.